spec/liquid_tags/golang_tag_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe GolangTag, type: :liquid_tag do
+  describe "#link" do
+    let(:valid_link) { "https://play.golang.org/p/HmnNoBf0p1z" }
+    let(:missing_link) { "https://play.golang.org/p/HmnNoBf0p1z" }
+
+    def generate_tag(link)
+      Liquid::Template.register_tag("golang", GolangTag)
+      Liquid::Template.parse("{% golang #{link} %}")
+    end
+
+    it "accepts valid input" do
+      expect { generate_tag(valid_link) }.not_to raise_error
+    end
+
+    it "renders valid input" do
+      template = generate_tag(valid_link)
+      expected = 'src="https://play.golang.org/p/HmnNoBf0p1z"'
+      expect(template.render(nil)).to include(expected)
+    end
+
+    it "accepts only Kotlin Playground links" do
```

Comment:
This probably shouldn't say "Kotlin Playground Links".

Reply:
I think Chuck meant to make the PR a draft PR. I've converted it to a draft as I know he was going to work on the specs after our pairing session.